src/CMakeLists.txt

```diff
@@ -83,6 +83,93 @@ set(SOURCES
     utils/pdu_utils.cpp
 )
 
+set(HEADERS
+    ${LIBTINS_INCLUDE_DIR}tins/address_range.h
```

Comment:
Can you add a slash before `tins` on all these paths? e.g. `${LIBTINS_INCLUDE_DIR}/tins/address_range.h`.

Reply:
Sure. I think that ends up with a double slash when concatentated with the variable but suspect CMake is agnostic to that.